add-jsdocs.mjs

```
@@ -0,0 +1,75 @@
+      if (fs.statSync(dirFile).isDirectory()) {
+        filelist = walkSync(dirFile, filelist);
+      } else {
+        if (
+          dirFile.endsWith('.ts') &&
+          !dirFile.includes('.test.') &&
+          !dirFile.includes('.d.ts')
+        ) {
+          filelist.push(dirFile);
+        }
+      }
+    } catch (e) {}
```

ESLint

[error] 'e' is defined but never used.

(`@typescript-eslint/no-unused-vars`)

---

[error] Empty block statement.

(no-empty)